x-pack/plugins/maps/public/classes/sources/es_pew_pew_source/es_pew_pew_source.js

```diff
@@ -16,6 +16,7 @@ import { convertToLines } from './convert_to_lines';
 import { AbstractESAggSource } from '../es_agg_source';
```

Comment:
Do we even want pew-pew source to support geo_shape geo_tile_grid aggregations? I think since the source is point to point it should not.

Reply:
geo_shape can contain points. Ideally, users would not have to know if they should be using `geo_point` or `geo_shape`. `geo_shape` should always be the safe default choise imho. I think this is fine, but can remove.

Reply:
discussed in person. removed support for pew-pew sources. There's some edge-cases with regard to how pewpew sources collapse source-fields that are not being handled. Will add in separate PR.